security_token.py

```diff
@@ -91,10 +91,9 @@ def approval_program():
     grant_roles = Seq([
         Assert(And(
             is_contract_admin,
-            Txn.accounts.length() == Int(1),
```

Comment:
Had to remove some code to to accommodate the 1000 byte limit. This appears to be one of the lowest risk changes.